scripts/update_rules.sh

```diff
@@ -4,6 +4,20 @@
 # TODO: Include steps for downloading/updating local.rules from a remote source
 
 # Run pulledpork.pl to download newest rules, process disabled rules, and save new ruleset.
+
+check_state() {
+    if [[ $(id -u) != "0" ]]; then
```

Comment:
Root check should go into it's own function. No reason to have it in check_state